src/main/java/games/strategy/engine/ClientFileSystemHelper.java

```diff
@@ -97,7 +99,10 @@ private static File getRootRelativeToClassFile(final String fileName) {
     return f;
   }
 
-  private static boolean folderContainsGamePropsFile(final File folder) {
+  private static boolean folderContainsGamePropsFile(File folder) {
+    if (!folder.isDirectory()) {
+      folder = new File(folder.toString().substring(5));
```

Comment:
What does the magic number "5" signify?  It's obviously the length of some path prefix, but I can't figure out what that is.